### DISCUSSION TITLE
Converting RPU to profile 10

### DISCUSSION THREAD

**Discussion by Boulder08** [quietvoid/dovi_tool#314] [Q&A] [answered]
I'm currently using mode 2 to convert the RPU from profile 7 to 8.1 to use it in a encode which includes using DoViBaker. It seems to work fine with my AV1 encodes, but is there something different in profile 10 so that a new mode should actually be requested?

Thanks!

**Comment by quietvoid** [comment, marked as answer]
The RPU stays the same.
For AV1 only the container metadata must be profile 10.
